website/docs: updated user count info (#14186)
goauthentik/authentik: website/docs/enterprise/manage-enterprise.mdx

### About users and licenses

> License usage is calculated based on total user counts that authentik regularly captures. This data is checked against all valid licenses, and the sum total of all users. Internal and external users are counted based on the total number of users of the respective type saved in authentik.

Reviewer: ```suggestion
License usage is determined by the total number of users that authentik regularly records. This information is compared to all active licenses and the overall user count. Both internal and external users are included, based on the total number of each type stored in authentik.
```

Author: thanks, I like that last bit, am taking it... :-)

:::info

> Disabled accounts and service account users are not counted towards the user count of the license.

Reviewer: ```suggestion
Accounts that are disabled, as well as service accounts, are excluded from the license user count.
```

Author: I almost didn't like it because it separates the words "disabled" and "accounts" but I like the rest of it so much so lgtm, thx. ;-)
